content/posts/team_intro.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Introducing the Team"
+description: "Introducing CNeuroMod Team Members"
+date: 2020-05-31 
+author: "Valentina Borghesani"
+image: "team_intro/team.png"
```

Comment:
The icon in the blog summary page appears black on dark grey. It would be better to have a color that pops, e.g. #36ad5d 

I have used red for the first blog, but in retrospect I should have stuck to the main color palette of the website.